hazelcast/src/main/java/com/hazelcast/map/impl/eviction/Evictor.java

```diff
@@ -32,20 +32,21 @@
      * @param recordStore the recordStore
      * @return removal size.
      */
-    int findRemovalSize(RecordStore recordStore);
+    int findSizeToEvict(RecordStore recordStore);
 
     /**
      * Remove supplied number of elements from record-store.
      *
-     * @param removalSize supplied size to remove.
+     * @param sizeToEvict supplied size to remove.
      * @param recordStore the recordStore
      */
-    void removeSize(int removalSize, RecordStore recordStore);
+    void evictBySize(int sizeToEvict, RecordStore recordStore);
```

Comment:
What does this mean? evict by size? And what does size mean? Is that item count or is that memory usage?